### PR TITLE
#164797979 User can edit profile

### PR DESCRIPTION
**Pull request by sojida**
#### Description
This PR implements user can edit profile

#### Type of change
- [x] New feature

#### How Has This Been Tested?

- [x] Unit testing

#### PT-ID

[#164797979](url)

#### Screenshots
N/A

#### Questions:
N/A

### CODE REVIEW

File: src/actions/profile.action.js [hunk 1]
```diff
@@ -58,7 +58,13 @@ const updateProfile = obj => {
...
       const user = decodeToken();
       const { data } = await http.patch(`/profile/${user.id}`, obj);
-      return dispatch(updateProfileSuccess(data));
+      dispatch(updateProfileSuccess(data));
+      toast.info('You have successfully update your profile.', {
+        type: toast.TYPE.INFO,
+        closeButton: false,
+        position: toast.POSITION.TOP_CENTER,
+      });
+      return 'success';
```
Comment (rotimi-babalola) [inline src/actions/profile.action.js]: You don't need to return anything from an action. Is this return value of `success` going to be used anywhere in the app? @sojida

File: src/actions/profile.action.js [hunk 2]
```diff
@@ -58,7 +58,13 @@ const updateProfile = obj => {
       }
       const user = decodeToken();
       const { data } = await http.patch(`/profile/${user.id}`, obj);
-      return dispatch(updateProfileSuccess(data));
+      dispatch(updateProfileSuccess(data));
+      toast.info('You have successfully update your profile.', {
```
Comment (vic3king) [inline src/actions/profile.action.js]: please use a more descriptive error message e.g `Profile update Successful`

File: src/components/presentations/Profilepage/__test__/Profilepage.actions.test.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
+/* eslint-disable no-dupe-keys */
```
Comment (vic3king) [inline src/components/presentations/Profilepage/__test__/Profilepage.actions.test.js]: @sojida  any reason for disabling this rule?

Comment (sojida) [reply src/components/presentations/Profilepage/__test__/Profilepage.actions.test.js]: I'll take it off. That's an oversight from my end.

File: src/actions/profile.action.js (hunk 2 above)
Comment (sojida) [reply src/actions/profile.action.js]: I'll update that.

File: src/actions/profile.action.js (hunk 1 above)
Comment (sojida) [reply src/actions/profile.action.js]: We are not using `success` anywhere. I'll remove it. Thanks

**Pull request merged**